roles/atomic_scan/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+# vim: set ft=ansible:
+#
+#   This role verifies whether container image doesn't exist CVEs.
+#
+- name: Fail if image is undefined
+  fail:
+    msg: "Image is undefined"
+  when: image is undefined
+
+- name: Modify image name in /etc/atomic.d/openscap
```

Comment:
I don't think this is necessary any more.  See [RHBZ#1418464](https://bugzilla.redhat.com/show_bug.cgi?id=1418464)

We should make the same change in [roles/atomic_scan_verify/tasks/main.yml](https://github.com/projectatomic/atomic-host-tests/blob/master/roles/atomic_scan_verify/tasks/main.yml#L38)